### ISSUE TITLE
Implicit dimension choice for log_softmax has been deprecated

### ISSUE THREAD

**Issue by sharpwood** [suragnair/alpha-zero-general#5]
UserWarning: Implicit dimension choice for log_softmax has been deprecated. Change the call to include dim=X as an argument.
  return F.log_softmax(pi), F.tanh(v)

**Comment by suragnair** [comment]
Thanks. Added dimension to log_softmax. Pull latest.
